appconfig/settings/wdb.q

```diff
@@ -0,0 +1,5 @@
+// Bespoke WDB settings for AWS finspace Starter Pack
+\d .wdb
+replay:0b;    //set replay to false so that replay isnt carried out during intial subscribe process
```

Comment:
Can we change this comment to be a more details "why?". And we don't need to comment "what" it is doing.

// We don't want to replay as the wdb will start before the period it is meant to subscribe to